vscode-react/src/components/aKDeploymentTableBody.component.tsx

```diff
@@ -116,6 +116,16 @@ export const AKDeploymentTableBody = ({ deployments }: { deployments?: Deploymen
 		hidePopper();
 	};
 
+	const deleteDeploymentDisplayPopper = (event: React.MouseEvent<HTMLDivElement>, deployment: Deployment) => {
+		if (deployment.state === DeploymentState.ACTIVE_DEPLOYMENT) {
```

Comment:
so I just click and nothing happens ?

Reply:
Yes, there is an indication with the mouse pointer that this action is not allowed, take a look here: https://tailwindcss.com/docs/cursor for `cursor-not-allowed`

Reply:
Done